src/pages/Unlock/Unlock.tsx

```diff
@@ -95,7 +108,20 @@ export const Unlock = () => {
       name: 'MultiversX Web Wallet',
       component: CrossWindowLoginButton,
     },
-  ];
+    {
+      name: 'Passkey Proxy',
```

Comment:
is mobile only?

Reply:
i have this filter to remove it from other screens
` {
      name: 'Passkey Proxy',
      component: IframeButton,
      loginType: IframeLoginTypes.passkey,
      onClick: () => onInitiateLogin(IframeLoginTypes.passkey),
      showOnlyOnMobile: true,
    },
    ...
    .filter((button) => !button.showOnlyOnMobile || width <= 768);
    `